### DISCUSSION TITLE
My Student Reverification Rejected

### DISCUSSION THREAD

**Discussion by jackyhjj** [community/community#61145] [GitHub Education]
### Hi everyone,

Hi, I have my school ID but it seems that the system auto rejects. Can any github staff or officer verify instead ? Thanks.

I have also submitted my ticket, but its been days with no response.

I need to keep using the perks from the pack and now I am having trouble accessing the perks and keep asking for my verification.

please somebody help!

**Comment by samus-aran** [comment]
Hi @jackyhjj  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by jackyhjj** [reply]
Thanks for the follow up, but I have used other way and got approved! Thanks !

another question is how long does it take to get the benefit after being approved ?

**Reply by riturajrawat** [reply]
Can you tell me what other way did you do to get approved?

**Reply by jackyhjj** [reply]
It turns out my photo of my school ID was too blurry, so i take the photo using my phone instead and it works. Hope it helps someone out there. Cheers!
